core/src/log.cpp

```diff
@@ -139,6 +147,33 @@ auto Logger::level() -> Logger::Level
     return state().level;
 }
 
+bool Logger::setFilePath(const std::string& filePath)
+{
+    if (!FileSystem::mount("/logs", std::make_unique<cubos::core::data::StandardArchive>("./logs", /*isDirectory=*/true,
+                                                                                         /*readOnly=*/false)))
+    {
+        return false;
+    }
+
+    auto fullPath = "/logs/" + filePath;
```

Comment:
I prefer to would leave this responsibility to allow more flexibility regarding the logs locations. Maybe rename this to `logToFile`, and provide another overload method `logToFile` which takes no arguments. That file could perform the base archive setup, choose a filename with the current timestamp (date and time), and then call the other `logToFile` overload.

You should also then add a call to `Logger::logToFile()` on the `Cubos` constructor.

What do you think?

Reply:
Does this mean we would need to have 2 different streams? What if both functions are called?

Reply:
No, one function is just a short-hand for calling the other. I think you should just `ASSERT` that a stream isn't already open - it doesn't make sense to call it twice.